beluga_ros/test/test_laser_scan.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright 2023 Ekumen, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gmock/gmock.h>
+#include <beluga_ros/laser_scan.hpp>
+
+#include <range/v3/range/conversion.hpp>
+
+#if BELUGA_ROS_VERSION == 1
+#include <boost/smart_ptr.hpp>
+#endif
+
+namespace {
+
+auto make_message() {
+#if BELUGA_ROS_VERSION == 2
+  return std::make_shared<beluga_ros::msg::LaserScan>();
+#elif BELUGA_ROS_VERSION == 1
+  return boost::make_shared<beluga_ros::msg::LaserScan>();
+#endif
+}
+
+TEST(TestLaserScan, MinMaxRangeFromMessage) {
+  auto message = make_message();
+  message->ranges = std::vector<float>{1., 2., 3.};
+  message->range_min = 10.f;
+  message->range_max = 100.f;
+  auto scan = beluga_ros::LaserScan(message);
+  ASSERT_EQ(scan.min_range(), 10.);
+  ASSERT_EQ(scan.max_range(), 100.);
+}
+
+TEST(TestLaserScan, MinMaxRangeFromConstructor) {
+  auto message = make_message();
+  message->range_min = 10.f;
+  message->range_max = 100.f;
+  auto scan = beluga_ros::LaserScan(message, Sophus::SE3d{}, 100UL, 15., 95.);
```

Comment:
@nahueespinosa nit^N: constants help readability:
```suggestion
  constexpr auto kOrigin = Sophus::SE3d{};
  constexpr auto kMaxBeams = 100UL;
  constexpr auto kMinRange = 15.;
  constexpr auto kMaxRange = 95.;
  auto scan = beluga_ros::LaserScan(message, kOrigin, kMaxBeams, kMinRange, kMaxRange);
```

Reply:
Fixed in https://github.com/Ekumen-OS/beluga/pull/271/commits/1af27f46170bf1712571bbb975ab65d95962cc46.